### PR TITLE
Feature/multiselect

### PR DESCRIPTION
**Pull request by j8seangel**
Includes 3 new components and refine some styles:

- Loading component
- Dropdown component
- Multiselect component
- Fix accordion styles
- Update docs

### CODE REVIEW

File: .eslintrc.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@ plugins: [react, prettier]
 rules:
   jsx-a11y/click-events-have-key-events: 'warn'
   jsx-a11y/no-noninteractive-element-interactions: 'warn'
+  import/extensions: ["error", "never", { "packages": "always" }]
```
Comment (aabdaab) [inline .eslintrc.yml]: I have added `import/extensions: [error, {"js": never, "jsx": never}]` as it is in climate watch, we can combine those two: 
`import/extensions: [error, {"js": never, "jsx": never, "packages": always}]`

File: src/components/dropdown/dropdown.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      disabled,
+      colorDot,
+      info,
+      infoText,
+      required,
+      optional,
+      value
+    } = this.props;
+    const themeIcon = white ? { icon: styles.white } : {};
+    const hasNotValue = value && !value.value;
```
Comment (aabdaab) [inline src/components/dropdown/dropdown.js]: please rename to `hasNoValue` for better reading

File: src/components/dropdown/dropdown.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+Dropdown.propTypes = {
+  label: PropTypes.string,
+  openUp: PropTypes.bool,
+  transparent: PropTypes.bool,
+  white: PropTypes.bool,
+  plain: PropTypes.bool,
+  dark: PropTypes.bool,
+  info: PropTypes.bool,
+  infoText: PropTypes.string,
+  theme: PropTypes.object,
```
Comment (aabdaab) [inline src/components/dropdown/dropdown.js]: I think we should be consistent and keep using `PropTypes.shape({...})` for defining what properties theme may have.

Overall, about the component; I know it's extracted but it just seems too crowded; following your proposition from yesterday about extensive styling options, maybe we should get rid of some? I would suggest getting rid of `white`, `plain`, `dark`, `blueBorder` and `colorDot` and replace them with theming, wdyt?

File: src/components/multiselect/multiselect-component.jsx [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+        </div>
+      </div>
+    );
+  }
+}
+
+MultiselectComponent.propTypes = {
+  values: PropTypes.array.isRequired,
+  options: PropTypes.array.isRequired,
+  theme: PropTypes.object,
```
Comment (aabdaab) [inline src/components/multiselect/multiselect-component.jsx]: same as above, wdyt about using `PropTypes.shape`?

File: src/components/dropdown/dropdown.js (hunk 3 above)
Comment (j8seangel) [reply src/components/dropdown/dropdown.js]: Agreed! I forgot to remove them thanks!
I'll move to the theme to keep the original functionality as examples.

File: .eslintrc.yml (hunk 1 above)
Comment (j8seangel) [reply .eslintrc.yml]: 👍

File: src/components/dropdown/dropdown.js (hunk 3 above)
Comment (j8seangel) [reply src/components/dropdown/dropdown.js]: Removed all of them except the withDot as it embed the feature of including the `dot` so renamed to withDot boolean and theme responsability to update the color.

**Pull request merged**